### ISSUE TITLE
Not work with GitLab repo

### ISSUE THREAD

**Issue by artemklevtsov** [ropensci/codemetar#245]
Hi,

Seem not work with gitlab.

``` r
> codemetar::create_codemeta(".")
Error in object[[name, exact = TRUE]] : subscript out of bounds
> traceback()
20: getElement(., 1)
19: function_list[[i]](value)
18: freduce(value, `_function_list`)
17: `_fseq`(`_lhs`)
16: eval(quote(`_fseq`(`_lhs`)), env, env)
15: eval(quote(`_fseq`(`_lhs`)), env, env)
14: withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
13: root %>% git2r::repository(discover = TRUE) %>% remote_urls() %>% 
        grep(pattern = "github", value = TRUE) %>% getElement(1) %>% 
        gsub(pattern = "git@github.com:", replacement = "https://github.com/") %>% 
        gsub(pattern = "\\.git$", replacement = "")
12: guess_github(root)
11: stri_replace_first_regex(string, pattern, fix_replacement(replacement), 
        opts_regex = opts(pattern))
10: str_replace(string, pattern, "")
9: stringr::str_remove(guess_github(root), ".*com\\/")
8: guess_readme_url(root)
7: `_f`(root = root)
6: withVisible(`_f`(root = root))
5: guess_readme(dirname(description_file))
4: try_to_give_opinions_readme(description_file)
3: do.call(rbind, list(give_opinions_desc(description = description), 
       try_to_give_opinions_readme(description_file)))
2: give_opinions(root)
1: codemetar::create_codemeta(".")
```

Package version is 0.1.8.

**Comment by maelle** [comment]
Hi @artemklevtsov, thanks for the bug report! If the repo is public, could you please paste the link so I can clone it and work on a fix? Thanks!

**Comment by artemklevtsov** [comment]
Sure. For example: https://gitlab.com/artemklevtsov/benchr or https://gitlab.com/artemklevtsov/uchardet.

**Comment by maelle** [comment]
Thank you!

**Comment by maelle** [comment]
I'll have a look next week, sorry for the delay!

**Comment by maelle** [comment]
@artemklevtsov can you try using the "repos" branch?

`remotes::install_github("ropensci/codemetar", ref = "repos")`

**Comment by maelle** [comment]
oops now wait a bit.

**Comment by maelle** [comment]
Now you can try it and tell me whether it solved your issues. Thanks for your patience!

**Comment by artemklevtsov** [comment]
Now it works. Result file: [codemeta.txt](https://github.com/ropensci/codemetar/files/3886392/codemeta.txt)

**Comment by maelle** [comment]
Awesome, great news, thanks again for the bug report and your patience!

**Comment by maelle** [comment]
Fixed in dev by #253
